docs/integration-guides/build-options.md

```diff
@@ -116,6 +116,7 @@ Format: `cmake -D VARNAME=VARVALUE`
 * `NANO_SECURE_RPC=ON` (to build node with TLS)
 * `NANO_WARN_TO_ERR=ON` (*v20.0+* turn compiler warnings into errors on Linux/Mac) 
 * `NANO_TIMED_LOCKS=50` (*v20.0+* when the number of milliseconds a mutex is held is equal or greater than this output a stacktrace, 0 disables. If enabled requires the boost `context` & `fiber` libraries)
+* `NANO_STACKTRACE_BACKTRACE=ON` (*v20.0+* uses Boost backtrace in stacktraces, to display filenames, function names and line numbers. Needs `libbacktrace` to be installed. Some [workarounds](https://www.boost.org/doc/libs/develop/doc/html/stacktrace/configuration_and_build.html#stacktrace.configuration_and_build.f3) may be necessary depending on system and configuration. Use CLI [`--debug_stacktrace`](/commands/command-line-interface#-debug_stacktrace) to get an example output.)
```

Comment:
We already use boost backtrace, this command is to use a different configuration so that the addresses may output more human readable information (like line numbers) on linux.